resolver.py

```diff
@@ -314,3 +314,179 @@ def get_references_and_repos_from_adfs(adfs):
+                                                                 projection={UniRegConstants.UID: 1}, find_one=True)
+                        else:
+                            _adf['document_number'] = _adf['document_number'].replace('–', '-')
+                            ret_val, document = DB.get_documents(UniRegConstants.COLLECTION,
+                                                                 query={UniRegConstants.DOCUMENT_NUMBER: _adf.get('document_number'),
+                                                                        UniRegConstants.REPO: repo},
+                                                                 projection={UniRegConstants.UID: 1}, find_one=True)
+
+                        if "uid_fields" not in copy_frames[index]:
+                            copy_frames[index]["uid_fields"] = []
+                        dict_regex = dict()
+                        dict_regex['repo'] = repo
+                        dict_regex['UID'] = []
+
+                        if ret_val is False:
+                            logger.warning("Failed to resolve adf: {0}".format(adf))
+                            dict_regex['UID'] = []
+
+                        if document:
+                            dict_regex['UID'] = [document.get(UniRegConstants.UID)]
+                        copy_frames[index]["uid_fields"].append(dict_regex)
+
+                    else:
+                        try:
+                            uid_regex = adf_resolver.get_uid_regex(_adf)
+                            # uid_regex = homoglyph_resolver(uid_regex)
+                            if uid_regex is None:
+                                logger.warning("Uid regex is None for adf: {0}".format(adf_resolver.adf))
+
+                        except Exception as e:
+                            uid_regex = None
+                            logger.warning("Got exception while fetching uid regex. Error: {0}".format(str(e)))
+
+                        if repo not in unique_uid_regex.keys():
+                            unique_uid_regex[repo] = set()
+
+                        unique_uid_regex[repo].add(uid_regex)
+
+                        if "uid_fields" not in copy_frames[index]:
+                            copy_frames[index]["uid_fields"] = []
+
+                        dict_regex = dict()
+                        if uid_regex:
+                            uid_regex = uid_regex[1:len(uid_regex)-1]
+
+                        dict_regex['uid_regex'] = uid_regex
+                        dict_regex['repo'] = repo
+                        copy_frames[index]["uid_fields"].append(dict_regex)
+    return copy_frames, unique_uid_regex  
+
+
+def get_uid_frames(frames):
+    """ returns list of uids from frames"""
+    copy_frames = deepcopy(frames)
+    unique_uid_regex = {}
+    copy_frames, unique_uid_regex = get_uid_lists(copy_frames, unique_uid_regex)
+
+    for repo_field, regexes in unique_uid_regex.items():
+        char_limit = 30000
```

Comment:
@DIVYANSH0074 have `char_limit` as an argument to this function instead of simply passing 30000. Possibly a lower `char_limit` might work better for some repos.